src/actions/index.js

```diff
@@ -1,16 +1,22 @@
-import axios from 'axios';
+import axios from "axios";
+import { GET_PAGES, FETCH_CLUBS, MODAL_OPEN, MODAL_CLOSE } from "./actionTypes";
 
-import { GET_PAGES, MODAL_OPEN, MODAL_CLOSE } from './actionTypes';
-
-const api = 'https://dev.huoa.org/wp-json';
+const api = "https://dev.huoa.org/wp-json";
 
 export const getPages = () => async (dispatch) => {
   const response = await axios.get(`${api}/wp/v2/pages?page=1&per_page=100`);
-  const data = response.data.reduce((allData, { slug, title, acf }) => ({ ...allData, [slug]: { title: title.rendered, ...acf }}), {});
-
+  const data = response.data.reduce(
+    (allData, { slug, title, acf }) => ({...allData,[slug]: { title: title.rendered, ...acf },}),
+    {}
+  );
   dispatch({ type: GET_PAGES, payload: data });
 };
 
+export const getClubs = () => async (dispatch) => {
+  const response = await axios.get(`${api}/wp/v2/clubs`);
+  dispatch({ type: FETCH_CLUBS, payload: response.data });
+};
+
```

Comment:
This will need a small update to look like other api get requests - see this example: https://github.com/Hawaian-United-Okinawa-Association/huoa-react/blob/main/src/actions/index.js#L8